cli/internal/cmd/info/bin.go

```diff
@@ -0,0 +1,85 @@
+package info
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/vercel/turborepo/cli/internal/cmdutil"
+	"github.com/vercel/turborepo/cli/internal/config"
+
+	"github.com/mitchellh/cli"
+	"github.com/spf13/cobra"
+)
+
+// BinCommand is the structure for the bin command
+type BinCommand struct {
+	Config *config.Config
+	UI     *cli.ColoredUi
+}
+
+// Synopsis of run command
+func (c *BinCommand) Synopsis() string {
```

Comment:
Can we do this, and the `Help()` command below, in terms of the `cobra` command? You can see `link` for an example